package.json

```diff
@@ -157,6 +157,7 @@
     "react-router-dom": "^5.2.0",
     "react-test-renderer": "^17.0.1",
     "remark": "^13.0.0",
+    "resize-observer-polyfill": "^1.5.1",
```

Comment:
I'm continuing the discussion here

```suggestion

```

<img width="794" alt="Capture d’écran 2021-07-28 à 17 15 58" src="https://user-images.githubusercontent.com/3165635/127348415-ddf5ce0c-2963-428e-8872-8fca1d3b1eb5.png">

Where do you get this reference error?

Reply:
If I try to use the built-in ResizeObserver without importing that polyfill package and access `http://localhost:3000/components/masonry/`, I get the following error: 
![Screenshot 2021-07-28 at 16 29 33](https://user-images.githubusercontent.com/32841130/127350986-56c691b8-3599-4571-9508-fe9221697ee5.png)

Reply:
Seeing https://caniuse.com/?search=ResizeObserver it should work :\ I think the `yarn.lock` is not pushed.

Reply:
Let's add it in the `packages/material-ui-lab/package.json`

Reply:
@hbjORb The error's stack trace shows that it's coming from the server. The root cause is that you can't use it server-side, there is no layout.

Reply:
That totally makes sense. I deleted the polyfill package and we now use the built-in ResizeObserver.